docs/source/deploy_chaincode.md

```diff
@@ -0,0 +1,348 @@
+```
+CC_PACKAGE_ID=fabcar_1:69de748301770f6ef64b42aa6bb6cb291df20aa39542c3ef94008615704007f3
+```
+
+Because you set the environment variables to operate the `peer` CLI as the Org2 admin, we can approve the chaincode definition of Fabcar for Org2. Chaincode is approved at the organization level, so the command only needs to target one peer. The approval is distributed to peers within each organization using gossip.
+```
+peer lifecycle chaincode approveformyorg -o localhost:7050 --ordererTLSHostnameOverride orderer.example.com --channelID mychannel --name fabcar --version 1.0 --init-required --package-id $CC_PACKAGE_ID --sequence 1 --tls true --cafile ${PWD}/organizations/ordererOrganizations/example.com/orderers/orderer.example.com/msp/tlscacerts/tlsca.example.com-cert.pem
+```
+
+The command above uses the `--package-id` flag to include the package identifier in the chaincode definition. The command also uses the `--init-required` flag to request that Init function be invoked initialize the chaincode, before other functions can be used to interact with the ledger. By default, the Init function is not required and cannot be executed. You do not need to request that the Init function be invoked to initialize your chaincode.
+
+We could have provided a ``--signature-policy`` or ``--channel-config-policy`` argument to the chaincode approval to specify a chaincode endorsement policy. The endorsement policy specifies how many peers belonging to different channel members need to validate a transaction against a given chaincode. Because we did not set a policy, the definition of Fabcar will use the default endorsement policy, which requires that a transaction be endorsed by a majority of channel members present when the transaction is submitted. This implies that if new organizations are added to or removed from the channel, the endorsement policy
+is updated automatically to require more or fewer endorsements. In this tutorial, the default policy will require a majority of 2 out of 2 and transactions will need to be endorsed by a peer from Org1 and Org2. If you want to specify a custom endorsement policy, you can use the [Endorsement Policies](endorsement-policies.html) operations guide to learn about the policy syntax.
+
+The `peer lifecycle chaincode approveformyorg` command needs to be submitted and signed as your organization admin identity. As a result, the `CORE_PEER_MSPCONFIGPATH` variable needs to point to the MSP folder of an admin identity. You cannot approve a chaincode definition with a client user. The approval needs to be submitted to the ordering service, which will validate the admin signature and then distribute the approval to your peers.
```

Comment:
Suggestion for clarity: You cannot approve a chaincode definition with an identity that does not have the admin role.

Reply:
done.